infra/infra-integrations/infra-integrations-email/src/main/java/com/sflpro/notifier/externalclients/email/mandrill/MandrillConfiguration.java

```diff
@@ -0,0 +1,45 @@
+package com.sflpro.notifier.externalclients.email.mandrill;
+
+import com.microtripit.mandrillapp.lutung.controller.MandrillMessagesApi;
+import com.sflpro.notifier.email.TemplatedEmailSenderRegistry;
+import com.sflpro.notifier.externalclients.email.mandrill.communicator.MandrillApiCommunicator;
+import com.sflpro.notifier.externalclients.email.mandrill.communicator.MandrillApiCommunicatorImpl;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.boot.autoconfigure.condition.ConditionalOnProperty;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.util.Assert;
+
+
+/**
+ * Created by Hayk Mkrtchyan.
+ * Date: 6/19/19
+ * Time: 11:39 AM
+ */
+@Configuration
+@ConditionalOnProperty(name = MandrillConfiguration.TOKEN_PLACE_HOLDER)
```

Comment:
I think we should use something programmatic instead of this. If we do it like this turning providers on and off becomes different for each of the providers. I think we should, in the future, opt for configuration propertiessimilar to the example below.
```
plugin.mandrill.enabled=true
plugin.smtp.enabled=true
```
or
```
plugins.enabled=mandril,smtp
```
Whichever ay we decide to go we'll have to do it programmatically and not spring configuration based.

Reply:
Done!